newdle/templates/joined_email.txt

```diff
@@ -0,0 +1,10 @@
+Hi,
+
+{{ participant }} replied to "{{ title }}".
+
+Selected dates:
+{% for item in answers %}
+    {{ item }}
+{% endfor %}
```

Comment:
I think you get some ugly extra linebreaks here. `{% for item in answers -%}` may be enough to avoid it, but you need to test this ;)

Reply:
@ThiefMaster 
```
{% for item in answers %-}
    {{ item }}
{% endfor %-}
```
Returns
```
  <p>Selected dates:</p>
  <ul>
    <li>Date: 18/09/2020, 10:00:00 (Available)</li>
    <li>Date: 18/09/2020, 13:30:00 (If needed)</li>
    </ul>
  <p>
```
Notice the ul closing tag displaced.

While
```
{% for item in answers %-}
    {{ item }}
{% endfor %}
```

Converts into 
```
  <p>Selected dates:</p>
  <ul>
    <li>Date: 18/09/2020, 10:00:00 (Available)</li>
    <li>Date: 18/09/2020, 13:30:00 (Available)</li>

  </ul>
  <p>
```

Notice the extra line. Some ideas?

Reply:
for the HTML version it doesn't matter, since linebreaks are pretty much meaningless in HTML.

only the plaintext version is where all whitespace is significant.

Reply:
It is good something like this?

```
{% for item in answers -%}
    • {{ item }}
{% endfor %}
```

returning

> Selected dates:
> • Date: 18/09/2020, 10:00:00 (If needed)
> • Date: 18/09/2020, 13:30:00 (Available)
> 
> Check all results on the summary page:
> ...../newdle/XXXXXX/summary

Reply:
I think I'd prefer simple dashes (`-`), but having some kind of list item makes sense for sure